CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # Changelog
 
+## v0.8.0
+
+* `nerves_hub_cli -> 0.10.0` - This decouples the CLI from the deprecated `:nerves_hub`
+lib and frees this lib to do the same. If you are setting `:nerves_hub, org: org` in
+your config, compilation will fail until you change it to use this lib key:
+```elixir
+config :nerves_hub_link, org: org
+```
+* `nerves_hub_user_api -> 0.6.0`
+
+* Enhancements
+  * Updates example app
+  * Cleanup and structural changes
+  * Changes where default NervesHub certs are stored and how they are read in
+
+* Fixes
+  * Fixes an issue where a device may get an update message from the server while
+  performing an update which would cause things to crash.
+  * Fixes path to provisioning.conf
+  * decouples `:nerves_hub` config values - see note above
```

Comment:
I think the only real change is the decoupling of config values. Weren't these other changes fixes to unreleased code?

Reply:
The path to provisioning conf and the issue where device can update while doing an update exist in 0.7.6 which is in hex